server/routes/clubs.js

```diff
@@ -22,7 +22,7 @@ router.post('/delete', (req, res) => {
 });
 
 /* /:id routes */
-router.get('/:id', (req, res) => {
+router.get('/:id', clubController.getClub,(req, res) => {
```

Comment:
Happy to flesh this route out a bit more if we want to get all the club info in one request.

Also if this is a protected route it should be placed behind a verifyToken controller.